src/main/resources/application-search.yml

```diff
@@ -26,3 +26,12 @@ mod-linked-data:
   kafka:
     topic:
       bibframe-index: search.bibframe
+
+folio:
+  kafka:
+    retry-interval-ms: ${KAFKA_RETRY_INTERVAL_MS:2000}
+    retry-delivery-attempts: ${KAFKA_RETRY_DELIVERY_ATTEMPTS:6}
+    topics:
```

Comment:
There is a problem with having the configuration in application-search.yml file.
For integrating with Inventory, we need to create inventory.instance_ingress topic. As this topic is not related to "search" functionality, we will be adding the configuration for inventory.instance_ingress in application.yml file.

That will cause problems as spring will not merge the list of topics from 2 yaml files.

May be get rid of "search" profile and use just "folio" profile?

Reply:
I remember related discussion with the LC: at some point they asked for Linked Data Editor to be able to deployed outside of folio but supporting search functionality (with substitution of mod-search with their own version implementing the same API). I am not sure if this is still actual requirement. Let's ask Doug?

Reply:
Anyway we could keep it as is for this PR and resolve during instance-ingress topic addition :)